From 715ae685756755ff290b0984d1ba8c74667a317f Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Wed, 22 Feb 2023 16:01:40 +0900
Subject: [PATCH] Fix sepolicy for offline charging EMUI9 kirin

---
 private/charger.te | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/private/charger.te b/private/charger.te
index c5f3a50a5..e212530ad 100644
--- a/private/charger.te
+++ b/private/charger.te
@@ -18,3 +18,17 @@ compatible_property_only(`
         -charger
     } charger_prop:file no_rw_file_perms;
 ')
+
+
+# Allow charger to write to sysfs_backlight_attr
+allow charger sysfs_backlight_attr:file w_file_perms;
+
+# Allow charger to write to sysfs_led_attr
+allow charger sysfs_led_attr:file w_file_perms;
+
+# Allow charger to read and write to sysfs_power
+allow charger sysfs_power:file rw_file_perms;
+
+# Huawei charger sysfs attr
+attribute sysfs_backlight_attr;
+attribute sysfs_led_attr;
\ No newline at end of file
-- 
2.34.1

