From c144a1736fa0a036bcb81f9b9a5056c845f906d6 Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Wed, 17 Aug 2022 12:52:41 +0900
Subject: [PATCH 8/8] fix:error found duplicate sysprop assignments:

---
 base.mk      | 1 -
 rw-system.sh | 1 -
 2 files changed, 2 deletions(-)

diff --git a/base.mk b/base.mk
index 83c0521..a15836c 100644
--- a/base.mk
+++ b/base.mk
@@ -24,7 +24,6 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
 	ro.build.version.all_codenames=$(PLATFORM_VERSION_ALL_CODENAMES) \
 	ro.build.version.release=$(PLATFORM_VERSION) \
 	ro.build.version.security_patch=$(PLATFORM_SECURITY_PATCH) \
-	ro.adb.secure=0 \
 	ro.logd.auditd=true
 	
 #Huawei HiSuite (also other OEM custom programs I guess) it's of no use in AOSP builds
diff --git a/rw-system.sh b/rw-system.sh
index 396312f..9f80787 100644
--- a/rw-system.sh
+++ b/rw-system.sh
@@ -766,7 +766,6 @@ if [ -f /system/phh/secure ] || [ -f /metadata/phh/secure ];then
     resetprop_phh ro.build.type user
     resetprop_phh ro.build.selinux 0
 
-    resetprop_phh ro.adb.secure 1
     setprop ctl.restart adbd
 
     # Hide system/xbin/su
-- 
2.37.1.windows.1

