From 2fcbebc71a35fdc322f00658cc1939e8a6db31da Mon Sep 17 00:00:00 2001
From: android <android>
Date: Wed, 28 Sep 2022 16:44:22 +0900
Subject: [PATCH 1/4] build: Bypass vendor calls

---
 target/product/aosp_arm64.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/target/product/aosp_arm64.mk b/target/product/aosp_arm64.mk
index 38f82a2d6..1b24fc647 100644
--- a/target/product/aosp_arm64.mk
+++ b/target/product/aosp_arm64.mk
@@ -51,8 +51,8 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/aosp_product.mk)
 #
 # All components inherited here go to vendor or vendor_boot image
 #
-$(call inherit-product, $(SRC_TARGET_DIR)/product/emulator_vendor.mk)
-$(call inherit-product, $(SRC_TARGET_DIR)/board/generic_arm64/device.mk)
+#$(call inherit-product, $(SRC_TARGET_DIR)/product/emulator_vendor.mk)
+#$(call inherit-product, $(SRC_TARGET_DIR)/board/generic_arm64/device.mk)
 
 #
 # Special settings for GSI releasing
-- 
2.34.1

