From 9b41bf3ef06186d38414137bc3128415734e8a12 Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Fri, 19 Aug 2022 22:56:59 +0900
Subject: [PATCH] ro.blaze.device err fix

---
 tools/buildinfo.sh | 1 -
 1 file changed, 1 deletion(-)

diff --git a/tools/buildinfo.sh b/tools/buildinfo.sh
index 69fbdda00..e604e48cf 100755
--- a/tools/buildinfo.sh
+++ b/tools/buildinfo.sh
@@ -58,6 +58,5 @@ if [ -n "$BUILD_THUMBPRINT" ] ; then
   echo "ro.build.thumbprint=$BUILD_THUMBPRINT"
 fi
 
-echo "ro.blaze.device=$BLAZE_DEVICE"
 
 echo "# end build properties"
-- 
2.36.1

