From 6990ade3dac6e2da08fbe274e7df8a258d61f713 Mon Sep 17 00:00:00 2001
From: android <android>
Date: Wed, 28 Sep 2022 16:49:56 +0900
Subject: [PATCH 3/4] Remove llkd

---
 target/product/base_system.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/target/product/base_system.mk b/target/product/base_system.mk
index 4ebc46432..f6877df8a 100644
--- a/target/product/base_system.mk
+++ b/target/product/base_system.mk
@@ -195,7 +195,6 @@ PRODUCT_PACKAGES += \
     libwilhelm \
     linker \
     linkerconfig \
-    llkd \
     lmkd \
     LocalTransport \
     locksettings \
-- 
2.34.1

