From dc27eaa68904da54c98d33e44fa0085893d10ee7 Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Sun, 14 Aug 2022 22:52:17 +0900
Subject: [PATCH 02/10] fix context

---
 sepolicy/file_contexts | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/sepolicy/file_contexts b/sepolicy/file_contexts
index 999ff97..aa0c8ef 100644
--- a/sepolicy/file_contexts
+++ b/sepolicy/file_contexts
@@ -7,8 +7,7 @@
 /system/bin/asus-motor u:object_r:phhsu_exec:s0
 /system/bin/xiaomi-touch u:object_r:phhsu_exec:s0
 
-#/system/bin/fsck\.exfat                 u:object_r:fsck_exec:s0
-/system/bin/fsck\.ntfs                  u:object_r:fsck_exec:s0
+
 
 /bt_firmware(/.*)?      u:object_r:bt_firmware_file:s0
 
-- 
2.37.1.windows.1

