From 19cf505bce8bce234bea92402fc66999e2dd84ef Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Mon, 15 Aug 2022 22:06:10 +0900
Subject: [PATCH 3/8] add libaptX

---
 generate.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/generate.sh b/generate.sh
index 5037834..cd1fa0b 100644
--- a/generate.sh
+++ b/generate.sh
@@ -94,7 +94,9 @@ PRODUCT_MODEL := Phh-Treble $apps_name
 # Overwrite the inherited "emulator" characteristics
 PRODUCT_CHARACTERISTICS := device
 
-PRODUCT_PACKAGES += $extra_packages
++PRODUCT_PACKAGES += $extra_packages \
++		ibaptX_encoder \
++    	libaptXHD_encoder
 
 EOF
 echo -e '\t$(LOCAL_DIR)/'$target.mk '\' >> AndroidProducts.mk
-- 
2.37.1.windows.1

