From 7d50165d0bf55608c5075b3a1c0246a6754b8e41 Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Mon, 15 Aug 2022 22:06:53 +0900
Subject: [PATCH 4/8] set OTA JSON URL

---
 generate.sh | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/generate.sh b/generate.sh
index cd1fa0b..67c7c78 100644
--- a/generate.sh
+++ b/generate.sh
@@ -95,8 +95,10 @@ PRODUCT_MODEL := Phh-Treble $apps_name
 PRODUCT_CHARACTERISTICS := device
 
 +PRODUCT_PACKAGES += $extra_packages \
-+		ibaptX_encoder \
-+    	libaptXHD_encoder
+		ibaptX_encoder \
+    	libaptXHD_encoder
+PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
+    ro.system.ota.json_url=https://raw.githubusercontent.com/j7b3y/treble_pb/master/ota.json
 
 EOF
 echo -e '\t$(LOCAL_DIR)/'$target.mk '\' >> AndroidProducts.mk
-- 
2.37.1.windows.1

