From 0b1674dc52608d5406bfbff20a4789cc90c4f4eb Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Sun, 14 Aug 2022 23:12:08 +0900
Subject: [PATCH 06/10] set OTA URL

---
 blaze.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/blaze.mk b/blaze.mk
index e82a3f8..6f26f40 100644
--- a/blaze.mk
+++ b/blaze.mk
@@ -1,3 +1,6 @@
 $(call inherit-product, vendor/blaze/config/common_full_phone.mk)
 -include vendor/blaze/build/core/config.mk
 -include vendor/blaze/build/core/apicheck.mk
+
+PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
+	ro.system.ota.json_url=https://raw.githubusercontent.com/j7b3y/treble_pb/master/ota.json
\ No newline at end of file
-- 
2.37.1.windows.1

