From 9e0586dc88931e565be2cbc0e8e92fcaa7550810 Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Mon, 15 Aug 2022 22:08:47 +0900
Subject: [PATCH 7/8] fix BOARD_KERNEL_IMAGE_NAME not defined

---
 board-base.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board-base.mk b/board-base.mk
index e363dd5..87cf710 100644
--- a/board-base.mk
+++ b/board-base.mk
@@ -9,3 +9,4 @@ BUILD_BROKEN_ELF_PREBUILT_PRODUCT_COPY_FILES := true
 BOARD_ROOT_EXTRA_SYMLINKS := $(filter-out $(BOARD_ROOT_EXTRA_SYMLINKS),/mnt/vendor/persist:/persist)
 
 BOARD_EXT4_SHARE_DUP_BLOCKS := true
+TARGET_NO_KERNEL_OVERRIDE := true
\ No newline at end of file
-- 
2.37.1.windows.1

