From 37c446f494b95e59393ac5e45b19b9a85e03dcbb Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Mon, 15 Aug 2022 21:25:19 +0900
Subject: [PATCH 809/809] set OTA JSON

---
 generate.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/generate.sh b/generate.sh
index 4a09a54..0e3a0f5 100644
--- a/generate.sh
+++ b/generate.sh
@@ -98,6 +98,9 @@ PRODUCT_PACKAGES += $extra_packages \
 		ibaptX_encoder \
     	libaptXHD_encoder
 
+PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
+    ro.system.ota.json_url=https://raw.githubusercontent.com/j7b3y/treble_pb/master/ota.json
+
 EOF
 echo -e '\t$(LOCAL_DIR)/'$target.mk '\' >> AndroidProducts.mk
 			done
-- 
2.36.1

