From f2ecb1bfc94cb07bd093c75ebeea8d883330e737 Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Fri, 19 Aug 2022 22:59:08 +0900
Subject: [PATCH 1/2] Disable inline kernel building

---
 config/common.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config/common.mk b/config/common.mk
index b001b6b2..44477bab 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -4,6 +4,7 @@ $(call inherit-product-if-exists, vendor/extra/product.mk)
 PRODUCT_BRAND ?= ProjectBlaze
 
 PRODUCT_BUILD_PROP_OVERRIDES += BUILD_UTC_DATE=0
+TARGET_NO_KERNEL_OVERRIDE := true
 
 ifeq ($(PRODUCT_GMS_CLIENTID_BASE),)
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
-- 
2.36.1

