From 13cac4f4c16e432d36cda69c394064dce0b83793 Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Mon, 15 Aug 2022 07:45:41 +0900
Subject: [PATCH 1/2] Disable inline kernel building

---
 config/common.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/common.mk b/config/common.mk
index c3512efa..38d82807 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -2,7 +2,7 @@
 $(call inherit-product-if-exists, vendor/extra/product.mk)
 
 PRODUCT_BRAND ?= ProjectBlaze
-
+TARGET_NO_KERNEL_OVERRIDE := true
 PRODUCT_BUILD_PROP_OVERRIDES += BUILD_UTC_DATE=0
 
 ifeq ($(PRODUCT_GMS_CLIENTID_BASE),)
-- 
2.36.1

