From 3188c2d4860701ae0456056475a27c029539e816 Mon Sep 17 00:00:00 2001
From: j7b3y <glowing3386@proton.me>
Date: Sat, 14 Oct 2023 22:58:43 +0900
Subject: [PATCH 2/3] config: Remove duplicate ro.adb.secure

---
 config/common.mk | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index edf9e008..f29b305d 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -18,10 +18,6 @@ ifeq ($(TARGET_BUILD_VARIANT),eng)
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.usb.config=adb
 else
-# Enable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.usb.config=none
-
 # Disable extra StrictMode features on all non-engineering builds
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.strictmode.disable=true
 endif
-- 
2.34.1

