From be84f1ee26a638b5eb310d244999abcbccff07e4 Mon Sep 17 00:00:00 2001
From: j7b3y <glowing3386@proton.me>
Date: Sat, 14 Oct 2023 22:35:29 +0900
Subject: [PATCH 2/3] config: Remove duplicate ro.adb.secure

---
 config/common.mk | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index edf9e008..1a98ea22 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -13,15 +13,6 @@ PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
     ro.com.google.clientidbase=$(PRODUCT_GMS_CLIENTID_BASE)
 endif
 
-ifeq ($(TARGET_BUILD_VARIANT),eng)
-# Disable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.usb.config=adb
-else
-# Enable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.usb.config=none
-
 # Disable extra StrictMode features on all non-engineering builds
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.strictmode.disable=true
 endif
-- 
2.34.1

