From 627c9ab07aa06aefbd7a97400f7e0d7ab5584c4d Mon Sep 17 00:00:00 2001
From: android <android>
Date: Wed, 28 Sep 2022 16:55:10 +0900
Subject: [PATCH 2/3] Remove duplicate ro.adb.secure

---
 config/common.mk | 2 --
 1 file changed, 2 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index 46cb2717..fe713c47 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -17,8 +17,6 @@ ifeq ($(TARGET_BUILD_VARIANT),eng)
 # Disable ADB authentication
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
 else
-# Enable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
 
 # Disable extra StrictMode features on all non-engineering builds
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.strictmode.disable=true
-- 
2.34.1

