From 81ad83b6a6c5c77bccccf742b9abef3b493a5804 Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Wed, 22 Feb 2023 16:04:20 +0900
Subject: [PATCH 2/4] onfig: Remove duplicate ro.adb.secure

---
 config/common.mk | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/config/common.mk b/config/common.mk
index edf9e008..9f63717e 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -18,9 +18,6 @@ ifeq ($(TARGET_BUILD_VARIANT),eng)
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=0
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.usb.config=adb
 else
-# Enable ADB authentication
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += ro.adb.secure=1
-PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.usb.config=none
 
 # Disable extra StrictMode features on all non-engineering builds
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += persist.sys.strictmode.disable=true
-- 
2.34.1

