From 3a955234ea7296e1973c3cdf97161fbf26b9a8fa Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Wed, 22 Feb 2023 16:05:50 +0900
Subject: [PATCH 3/4] build: Disable inline kernel building

---
 config/common.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/config/common.mk b/config/common.mk
index 4a8bff0c..5f51f321 100644
--- a/config/common.mk
+++ b/config/common.mk
@@ -97,6 +97,8 @@ PRODUCT_COPY_FILES += \
 PRODUCT_PRODUCT_PROPERTIES += \
     ro.boot.vendor.overlay.theme=com.android.internal.systemui.navbar.gestural;com.google.android.systemui.gxoverlay
 
+TARGET_NO_KERNEL_OVERRIDE := true
+
 # Enable IORap I/O Prefetching
 PRODUCT_SYSTEM_EXT_PROPERTIES += \
     persist.device_config.runtime_native_boot.iorap_perfetto_enable=true
-- 
2.34.1

