From 3e61cfd0a33aa8088a41024635315b26357eb3a6 Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Wed, 17 Aug 2022 12:46:53 +0900
Subject: [PATCH 1/8] Bye APN Config

---
 base-pre.mk | 4 +---
 base.mk     | 4 ----
 2 files changed, 1 insertion(+), 7 deletions(-)

diff --git a/base-pre.mk b/base-pre.mk
index 6a317e4..a7a3e3a 100644
--- a/base-pre.mk
+++ b/base-pre.mk
@@ -1,3 +1 @@
-#Use a more decent APN config
-PRODUCT_COPY_FILES += \
-	device/sample/etc/apns-full-conf.xml:system/etc/apns-conf.xml
+#Use a more decent APN config
\ No newline at end of file
diff --git a/base.mk b/base.mk
index af27de5..83c0521 100644
--- a/base.mk
+++ b/base.mk
@@ -8,10 +8,6 @@ PRODUCT_COPY_FILES := \
 	frameworks/native/data/etc/android.hardware.bluetooth_le.xml:system/etc/permissions/android.hardware.bluetooth_le.xml \
 	frameworks/native/data/etc/android.hardware.usb.host.xml:system/etc/permissions/android.hardware.usb.host.xml \
 
-#Use a more decent APN config
-PRODUCT_COPY_FILES += \
-	device/sample/etc/apns-full-conf.xml:system/etc/apns-conf.xml
-
 BOARD_PLAT_PRIVATE_SEPOLICY_DIR += device/phh/treble/sepolicy
 PRODUCT_PACKAGE_OVERLAYS += device/phh/treble/overlay
 
-- 
2.37.1.windows.1

