From 8aacd4accc0d5237143c48c490007ad6868aa0f4 Mon Sep 17 00:00:00 2001
From: android <android>
Date: Wed, 28 Sep 2022 16:25:46 +0900
Subject: [PATCH 01/12] Set BOARD_EXT4_SHARE_DUP_BLOCKS explicitly

---
 board-base.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board-base.mk b/board-base.mk
index 1ddacaf..0fe100e 100644
--- a/board-base.mk
+++ b/board-base.mk
@@ -7,3 +7,4 @@ BOARD_ROOT_EXTRA_FOLDERS += bt_firmware sec_storage efs persist
 BUILD_BROKEN_ELF_PREBUILT_PRODUCT_COPY_FILES := true
 
 BOARD_ROOT_EXTRA_SYMLINKS := $(filter-out $(BOARD_ROOT_EXTRA_SYMLINKS),/mnt/vendor/persist:/persist)
+BOARD_EXT4_SHARE_DUP_BLOCKS := true
-- 
2.34.1

