From a057d380b963078d22c3d1f15819946dd06e7d41 Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Wed, 22 Feb 2023 15:20:44 +0900
Subject: [PATCH 03/10] base: Provide libnfc-nci

---
 base.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/base.mk b/base.mk
index d076c69..b943940 100644
--- a/base.mk
+++ b/base.mk
@@ -56,6 +56,7 @@ PRODUCT_COPY_FILES += \
 #   Provide default libnfc-nci.conf file for devices that does not have one in
 #   vendor/etc
 PRODUCT_COPY_FILES += \
+	system/nfc/conf/libnfc-nci.conf:system/etc/libnfc-nci.conf \
 	device/phh/treble/nfc/libnfc-nci.conf:system/phh/libnfc-nci-oreo.conf \
 	device/phh/treble/nfc/libnfc-nci-huawei.conf:system/phh/libnfc-nci-huawei.conf
 
-- 
2.34.1

