From 477155be006edf5852daad8866484b4041d8bae7 Mon Sep 17 00:00:00 2001
From: android <android>
Date: Wed, 28 Sep 2022 16:28:49 +0900
Subject: [PATCH 05/12] Revert "Include sim toolkit app"

---
 base.mk | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/base.mk b/base.mk
index d3a0a20..180bfc4 100644
--- a/base.mk
+++ b/base.mk
@@ -160,8 +160,7 @@ PRODUCT_PACKAGES += \
 	xiaomi-motor \
 	oneplus-motor
 
-PRODUCT_PACKAGES += \
-	Stk
+
 
 PRODUCT_PACKAGES += \
 	resetprop_phh
-- 
2.34.1

