From b7e5510b59d2652fb74d28d961d3bedfcc944841 Mon Sep 17 00:00:00 2001
From: android <android>
Date: Wed, 28 Sep 2022 16:31:02 +0900
Subject: [PATCH 06/12] Bye APN Config

---
 base-pre.mk | 4 +---
 base.mk     | 3 ---
 2 files changed, 1 insertion(+), 6 deletions(-)

diff --git a/base-pre.mk b/base-pre.mk
index 6a317e4..a7a3e3a 100644
--- a/base-pre.mk
+++ b/base-pre.mk
@@ -1,3 +1 @@
-#Use a more decent APN config
-PRODUCT_COPY_FILES += \
-	device/sample/etc/apns-full-conf.xml:system/etc/apns-conf.xml
+#Use a more decent APN config
\ No newline at end of file
diff --git a/base.mk b/base.mk
index 180bfc4..89b81d6 100644
--- a/base.mk
+++ b/base.mk
@@ -8,9 +8,6 @@ PRODUCT_COPY_FILES := \
 	frameworks/native/data/etc/android.hardware.bluetooth_le.xml:system/etc/permissions/android.hardware.bluetooth_le.xml \
 	frameworks/native/data/etc/android.hardware.usb.host.xml:system/etc/permissions/android.hardware.usb.host.xml \
 
-#Use a more decent APN config
-PRODUCT_COPY_FILES += \
-	device/sample/etc/apns-full-conf.xml:system/etc/apns-conf.xml
 
 BOARD_PLAT_PRIVATE_SEPOLICY_DIR += device/phh/treble/sepolicy
 PRODUCT_PACKAGE_OVERLAYS += device/phh/treble/overlay
-- 
2.34.1

