From 38aee4ae4acdc6e56959ed8d251b09b50493ab79 Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Wed, 22 Feb 2023 15:27:32 +0900
Subject: [PATCH 06/10] board: Set BOARD_EXT4_SHARE_DUP_BLOCKS explicitly

---
 board-base.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/board-base.mk b/board-base.mk
index c38e385..1bc806d 100644
--- a/board-base.mk
+++ b/board-base.mk
@@ -9,5 +9,7 @@ DEVICE_FRAMEWORK_MANIFEST_FILE := device/phh/treble/framework_manifest.xml
 BOARD_ROOT_EXTRA_FOLDERS += bt_firmware sec_storage efs persist
 BUILD_BROKEN_ELF_PREBUILT_PRODUCT_COPY_FILES := true
 BUILD_BROKEN_DUP_RULES := true
+BOARD_EXT4_SHARE_DUP_BLOCKS := true
+
 
 BOARD_ROOT_EXTRA_SYMLINKS := $(filter-out $(BOARD_ROOT_EXTRA_SYMLINKS),/mnt/vendor/persist:/persist)
-- 
2.34.1

