From 42a695105abeb1138009e3f13a692f2a095aa76d Mon Sep 17 00:00:00 2001
From: android <android>
Date: Wed, 28 Sep 2022 16:32:02 +0900
Subject: [PATCH 07/12] change target name

---
 generate.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/generate.sh b/generate.sh
index fac8208..5037834 100644
--- a/generate.sh
+++ b/generate.sh
@@ -64,7 +64,7 @@ for part in a ab;do
 					optional_base='$(call inherit-product, device/phh/treble/base-sas.mk)'
 				fi
 
-				target="treble_${arch}_${part_suffix}${apps_suffix}${su_suffix}"
+				target="blaze_${arch}_${part_suffix}${apps_suffix}${su_suffix}"
 
 				baseArch="$arch"
 				if [ "$arch" = "a64" ];then
-- 
2.34.1

