From 4f5276323ed8df6a41ae2e2a05ab59e66ad2b428 Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Wed, 22 Feb 2023 16:12:08 +0900
Subject: [PATCH 08/10] change target name

---
 generate.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/generate.sh b/generate.sh
index 9c20eb5..f62c025 100644
--- a/generate.sh
+++ b/generate.sh
@@ -64,7 +64,7 @@ for part in a ab;do
 					optional_base='$(call inherit-product, device/phh/treble/base-sas.mk)'
 				fi
 
-				target="treble_${arch}_${part_suffix}${apps_suffix}${su_suffix}"
+				target="blaze_${arch}_${part_suffix}${apps_suffix}${su_suffix}"
 
 				baseArch="$arch"
 				if [ "$arch" = "a64" ];then
-- 
2.34.1

