From c68bdfe04929f2911f4cf4c251e72e84cf979d05 Mon Sep 17 00:00:00 2001
From: android <android>
Date: Wed, 28 Sep 2022 16:32:52 +0900
Subject: [PATCH 08/12] fix context

---
 sepolicy/file_contexts | 2 --
 1 file changed, 2 deletions(-)

diff --git a/sepolicy/file_contexts b/sepolicy/file_contexts
index 999ff97..e02fc87 100644
--- a/sepolicy/file_contexts
+++ b/sepolicy/file_contexts
@@ -7,8 +7,6 @@
 /system/bin/asus-motor u:object_r:phhsu_exec:s0
 /system/bin/xiaomi-touch u:object_r:phhsu_exec:s0
 
-#/system/bin/fsck\.exfat                 u:object_r:fsck_exec:s0
-/system/bin/fsck\.ntfs                  u:object_r:fsck_exec:s0
 
 /bt_firmware(/.*)?      u:object_r:bt_firmware_file:s0
 
-- 
2.34.1

