From fcd3f22f05450f6fa2bfb2d45fc678c9c1a9368f Mon Sep 17 00:00:00 2001
From: android <android>
Date: Wed, 28 Sep 2022 16:34:19 +0900
Subject: [PATCH 10/12] Add blaze.mk

---
 blaze.mk | 11 +++++++++++
 1 file changed, 11 insertions(+)
 create mode 100644 blaze.mk

diff --git a/blaze.mk b/blaze.mk
new file mode 100644
index 0000000..91acd2c
--- /dev/null
+++ b/blaze.mk
@@ -0,0 +1,11 @@
+$(call inherit-product, vendor/blaze/config/common_full_phone.mk)
+-include vendor/blaze/build/core/config.mk
+-include vendor/blaze/build/core/apicheck.mk
+
+TARGET_BOOT_ANIMATION_RES := 1080
+BLAZE_MAINTAINER := j7b3y
+TARGET_SUPPORTS_QUICK_TAP := true
+TARGET_SUPPORTS_CALL_RECORDING := true
+PRODUCT_PACKAGES += \
+    libaptX_encoder \
+    libaptXHD_encoder
\ No newline at end of file
-- 
2.34.1

