From baa17260d4078ae022983cdaa0074527efe44ecf Mon Sep 17 00:00:00 2001
From: j7b3y <Glowing3386@proton.me>
Date: Thu, 23 Feb 2023 18:46:34 +0900
Subject: [PATCH 10/10] use pe sepolicy

---
 blaze.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/blaze.mk b/blaze.mk
index e5ec51e..9f45aef 100644
--- a/blaze.mk
+++ b/blaze.mk
@@ -1,6 +1,6 @@
 $(call inherit-product, vendor/blaze/config/common.mk)
 $(call inherit-product, vendor/blaze/config/BoardConfigSoong.mk)
-$(call inherit-product, device/blaze/sepolicy/common/sepolicy.mk)
+$(call inherit-product, device/pe/sepolicy/common/sepolicy.mk)
 -include vendor/blaze/build/core/config.mk
 -include vendor/blaze/build/core/apicheck.mk
 
-- 
2.34.1

